server.js

```diff
@@ -5590,6 +5590,57 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Swagger Validator integration.
+camp.route(/^\/swagger\/(v)\/(2\.0)\/(https?)\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var type = match[1];        // e.g. `v` for validate
+  var specVer = match[2];     // e.g. `2.0` for OpenAPI 2.0
+  var scheme = match[3];      // e.g. `https`
+  var swaggerUrl = match[4];  // e.g. `api.example.com/swagger.yaml`
+  var format = match[5];
+
+  var badgeData = getBadgeData('swagger', data);
+
+  var urlParam = encodeURIComponent(scheme + '://' + swaggerUrl);
+  var url = 'http://online.swagger.io/validator/debug?url=' + urlParam;
```

Comment:
Maybe.  There are some tradeoffs.  The certificate used by online.swagger.io is missing an intermediate certificate (swagger-api/validator-badge#98).  The [workaround that I used](https://github.com/kevinoid/swagger-spec-validator/blob/v1.0.0/index.js#L54) in the `swagger-spec-validator` package is to include the intermediate certificate in the package.  The code to use it is also non-trivial, unfortunately.  Do you think it's worth the effort?